peepingtom/io_/read/read.py

```diff
@@ -18,6 +18,7 @@
 readers = {
     ('.star',): (read_star,),
     ('.mrc',): (read_mrc,),
+    ('.map'): (read_mrc,),
```

Comment:
They eay I designed this, you should put the `.map` extension in the tuple together with `/.mrc`. Anything that matches one of the elements in that key tuple will try to use the readers on the value tuple in order!